src/utils/emoji.ts

```diff
@@ -0,0 +1,59 @@
+const U200D = String.fromCharCode(0x200d);
+const UFE0Fg = /\uFE0F/g;
+
+const REGION_EMOJI = [
+	'❓',
+	'🖥️', // 1
+	'🇳🇱',
+	'🇮🇳',
+	'🇩🇪',
+	'🇬🇧', // 5
+	'🇺🇸',
+	'🇺🇸',
+	'🇸🇬',
+	'🇨🇦',
+	'🇮🇳', // 10
+	'🇦🇺',
+	'🇺🇸',
+	'🇺🇸',
+	'❓',
+	'🇯🇵' // 15
+];
```

Comment:
This needs to be updated to match the numbers at https://github.com/rivet-gg/rivet/blob/main/proto/backend/region.proto#L25-L56. The issue with this is i doubt there is an emoji for each of those places, so for now we can just use the emoji of the country it is in.

Reply:
@MasterPtato could you add this enum to api sdk? it would help to keep track with regions and displaying them properly on the frontend

Reply:
That sounds like a good idea. I'll see if I can get `universal_region` to be an enum instead of a number. Will get back to you on that